config/initializers/sso.rb

```diff
@@ -0,0 +1,2 @@
+Rails.configuration.sso_admin_org = ENV.
+fetch('sso_admin_org', 'pvb.digital.headquarters.moj')
```

Comment:
The default should be `digital.noms.moj` – because PVB is only available for teams within the `noms.moj` organisation.